var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -481,12 +513,41 @@ def cmake_args(self):
                 'ON' if '+teuchos' in spec else 'OFF'),
             '-DTrilinos_ENABLE_Tpetra:BOOL=%s' % (
                 'ON' if '+tpetra' in spec else 'OFF'),
+            '-DTrilinos_ENABLE_TrilinosCouplings:BOOL=%s' % (
+                'ON' if '+couplings' in spec else 'OFF'),
             '-DTrilinos_ENABLE_Zoltan:BOOL=%s' % (
                 'ON' if '+zoltan' in spec else 'OFF'),
             '-DTrilinos_ENABLE_Zoltan2:BOOL=%s' % (
                 'ON' if '+zoltan2' in spec else 'OFF'),
         ])
 
+        if '+expat' in spec:
+            options.extend([
+                '-DTEUCHOS_ENABLE_expat:BOOL=ON'
```

Comment:
All this options should be changed to oneliners similar to above, and moved to `extend` aove. For example,
```
-DTEUCHOS_ENABLE_expat:BOOL=%s' % (
   'ON' if '+expat' in spec else 'OFF'),
```

Reply:
Done.